### ISSUE TITLE
Sites to visit

### ISSUE THREAD

**Issue by bensig** [cybercongress/cyb-archeology#248]
Where can I find sites / dapps to visit using your browser? There a few broken links in your readme.

**Comment by mastercyb** [comment]
Yep. While [we know for sure](https://www.youtube.com/watch?v=pJOG5Ql7ZD0) that there are a lot of content in web3 exist still there is no easy way to surf it. We hope to launch a search network in coming months, hence you and me will be able to find something
